.env.example

```diff
@@ -91,13 +91,21 @@ export BOOKWAREHOUSE_NAMESPACE=bookwarehouse
 # export ENABLE_EGRESS=true
 
 # optional: ENABLE_GRAFANA (true/false)
-# Default: false
+# Default: true
 # export ENABLE_GRAFANA=true
 
 # optional: ENABLE_FLUENTBIT (true/false)
 # Default: false
 # export ENABLE_FLUENTBIT=true
 
+# optional: ENABLE_PROMETHEUS (true/false)
+# Default: true
```

Comment:
Same as above, Also we should probably rename this to DEPLOY_PROMETHEUS, WDYT ?

Reply:
Renamed to ENABLE_PROMETHEUS_DEPLOYMENT to match enable grafana. I can change both flags to deploy_* in the future if that's what we want

Reply:
This should be good for now